### DISCUSSION TITLE
How we collaborate

### DISCUSSION THREAD

**Discussion by Samsonroyal** [Samsonroyal/clearpath_backend#12] [General]
Hello @francoismavunila and @ikeecode @saadatsamjo @AJKossi @paulocouana @Akweley @Surfskills this is where we have conversations from

**Comment by Akweley** [comment]
Well noted.

**Reply by Samsonroyal** [reply]
Great looking forward to having you on discussions @Akweley

**Comment by Surfskills** [comment]
The Google social auth is implemented @Samsonroyal

**Reply by Samsonroyal** [reply]
This is currently under review; @bright2kwame and @ikeecode were requested to review changes

**Comment by francoismavunila** [comment]
> Hello @francoismavunila and @ikeecode @saadatsamjo @AJKossi @paulocouana @Akweley @Surfskills this is where we have conversations from

schema diagrams for those who want to edit
https://lucid.app/lucidchart/abcd8842-cf57-425b-9ade-8f28ce5c0902/edit?viewport_loc=-436%2C-322%2C3532%2C1392%2CFSuIvIhI0O4s&invitationId=inv_adc1a529-5255-4be0-9281-50277faae9bb

[clear_path_db_model.pdf](https://github.com/user-attachments/files/15598165/clear_path_db_model.pdf)

**Reply by Samsonroyal** [reply]
Thanks @francoismavunila
